README.rst

```diff
@@ -14,9 +16,47 @@ The following GUI backends are supported:
 - PyQt
 - PySide
 
+**Warning:** In TraitsUI version 5.0 the default GUI backend will change from
```

Comment:
I know - see earlier revisions and the update to the docs.  Unfortunately Github doesn't render admonitions properly for the README on the project landing page, which is why I wanted this warning here.